package.json

```diff
@@ -44,7 +44,7 @@
     "prettier": "@baltpeter/prettier-config",
     "dependencies": {
         "cross-dirname": "^0.1.0",
-        "trackhar": "^0.1.1"
+        "trackhar": "^0.3.0"
```

Comment:
TrackHAR 0.3.0 brings quite a few new properties, reasonings and maybe other stuff that I'm not thinking of right now. Those need additional changes here in tracker-wiki. I've tried to always mention them in the PR descriptions.

![image](https://github.com/tweaselORG/tracker-wiki/assets/4048809/d1a2124d-03ee-4430-ba40-8d54852a8725)